Tokens/solana-dvp/README.md

```diff
@@ -0,0 +1,55 @@
+# Solana DvP Sample CorDapp 
+
+This CorDapp provides an example to perform a DvP (Delivery vs Payment) of an [Evolvable](https://training.corda.net/libraries/token-sdk/token-introduction/#evolvabletokentype), [NonFungible](https://training.corda.net/libraries/token-sdk/token-introduction/#nonfungibletoken) token in 
+Corda utilizing the [Token SDK](https://github.com/corda/token-sdk).
+
+## Concepts
+
+The DvP is atomic: two participants agree on the delivery of an asset and the payment for it. 
+The payment is executed on Solana by the Notary, acting on behalf of the payer.
+
+### Flows
+
+There are two flows that we'll primarily use in this example that you'll be building off of.
+
+Prerequisite: Solana account needs to be created and contains enough token amounts.
+
+1. Create and issue a Corda token using `TokenCreateAndIssueFlow`, the toke will be used for 'Delivery" in Delivery Versus Payment.
```

Comment:
I know you're using the Tokens SDK, but the use of the term "token" for the Corda state is going to be confusing to anyone reading this. From a Solana context they will assume it means Solana tokens. Let's not refer to the Corda states as tokens and just call them Corda state, or similar.